docs/source/theory.rst

```diff
@@ -272,9 +272,76 @@ Finally, convective heat fluxes can be applied to boundaries:
 
 where :math:`h` is the heat transfer coefficient and :math:`T_{\mathrm{ext}}` is the external temperature.
 
----------------
+Kinetic surface model
+^^^^^^^^^^^^^^^^^^^^^
+
+Modelling hydrogen retention or outgassing might require considering the kinetics of surface processes. 
+A representative example is the hydrogen uptake from a gas phase, when the energy of incident atoms/molecules is not high enough to 
+overcome the surface barrier for implantation. The general approach to account for surface kinetics :cite:`Pick1985, Hodille2017, Guterl2019, Schmid2021` consists in 
+introducing hydrogen surface species :math:`c_\mathrm{s}`. 
```

Comment:
We should specify the dimension of $c_\mathrm{s}$ since it differs from the bulk concentration

Reply:
The units for parameters are not listed at all. Possibly, it'd beneficial to add units to all quantities. What do you think?

Reply:
Yes I think readers would look for units here.